packages/theme/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "@next-core/theme",
+  "version": "0.0.0",
+  "license": "GPL-3.0",
+  "description": "> TODO: description",
+  "main": "./src/index.css"
```

新的包统一使用 Pure ES Module
```suggestion
  "type": "module",
  "exports": {
    ".": "./src/index.css",
    "./package.json": "./package.json"
  }
```